TimeTrackerApp/TimeTrackerApp/Utilites/Extensions/Font+Extension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Font+Extension.swift
+//  TimeTrackerApp
+//
+//  Created by Sevda Gul Baran on 13.09.2022.
+//
+
+import Foundation
+import UIKit
```

Comment:
UIKit zaten Foundation'ı barındırıyor gerek yok.

```suggestion
import UIKit
```